src/olympia/editors/tests/test_models.py

```diff
@@ -97,29 +37,32 @@ def create_search_ext(name, version_str, addon_status, file_status,
 class TestQueue(TestCase):
     """Tests common attributes and coercions that each view must support."""
     __test__ = False  # this is an abstract test case
-    listed = True  # Are we testing listed or unlisted queues?
 
     def test_latest_version(self):
-        self.new_file(version=u'0.1', created=self.days_ago(2))
-        self.new_file(version=u'0.2', created=self.days_ago(1))
-        self.new_file(version=u'0.3')
+        addon = self.new_addon()
+        v1 = addon.find_latest_version(self.channel)
+        v1.update(created=self.days_ago(2))
+        v1.all_files[0].update(status=amo.STATUS_PUBLIC)
+        version_factory(addon=addon, version='2.0', created=self.days_ago(1),
```

Comment:
it's handled in `addon_factory` and `version_factory`, though it'd work in `file_factory` anyway if you passed in `created` as a `file_kw` (though you don't get the `updated` field set for free)